operator/pkg/helmreconciler/reconciler.go

```diff
@@ -573,6 +578,31 @@ func (h *HelmReconciler) analyzeWebhooks(whs []string) error {
 	return nil
 }
 
+func validateManifests(manifests name.ManifestMap) error {
+	// Ztunnel cannot be installed with other components
+	ztunnelEnabled := false
+	totalEnabled := 0
+	for c, ms := range manifests {
+		disabled := false
+		for _, m := range ms {
+			if strings.Contains(m, component.DisabledStr) {
+				disabled = true
+				break
+			}
+		}
+		if !disabled {
+			totalEnabled++
+			if c == name.ZtunnelComponentName {
+				ztunnelEnabled = true
+			}
+		}
+	}
+	if ztunnelEnabled && totalEnabled > 1 {
+		return fmt.Errorf("%s cannot be installed with other components", name.UserFacingComponentName(name.ZtunnelComponentName))
```

Comment:
Might be more useful here to tell people to `helm install ztunnel`,  versus just refusing with no explanation or push towards the preferred approach.

Reply:
This sounds good.